MAUI/TabView/Getting-Started.md

```diff
@@ -210,4 +210,250 @@ namespace TabViewMauiSample
 
 ![Getting_Started](images/Getting_Started.png)
 
-N> View [sample](https://github.com/SyncfusionExamples/maui-tabview-samples/tree/main/TabViewGettingStarted) in GitHub
\ No newline at end of file
+N> View [sample](https://github.com/SyncfusionExamples/maui-tabview-samples/tree/main/TabViewGettingStarted) in GitHub
+
+## Populate ItemsSource
+
+Items can be added to the control using the `ItemsSource` property of [SfTabView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TabView.SfTabView.html).
+
+Using ItemsSource, objects of any class can be given as items for SfTabView. The views corresponding to the objects can be set using the HeaderItemTemplate for the header items and ContentItemTemplate for the content. 
+
+Create a model class with TabItems collection property initialized with required number of data objects   as shown in the following code example.
```

Comment:
Create a model class using the `TabItems` collection property that is initialized with the required number of data objects,   as shown in the following code example.